src/app/fdctl/run/tiles/fd_replay.c

```diff
@@ -686,7 +686,7 @@ read_snapshot( void * _ctx, char const * snapshotfile, char const * incremental
 
       /* Open the command for reading. */
       char cmd[128];
-      snprintf( cmd, sizeof( cmd ), "./download_incremental.sh %s", incremental );
+      snprintf( cmd, sizeof( cmd ), "./contrib/download_incremental.sh %s", incremental );
```

Comment:
We should think about a better way of finding this script... like a environmental variable or something?